en/Advanced topics/Insider builds.md

```diff
@@ -39,3 +39,18 @@ To report an issue, please use one of the following channels:
 - In the forum, create a new topic under [Bug reports](https://forum.obsidian.md/c/bug-reports/7).
 
 When you report an issue, include the build version and the OS you're running it on. You can find the build version under **Settings > About > App > Current version**.
+
+## Go back to public builds on desktop
+
+To go back to using the public (non-Insider) builds on desktop:
+
+1. Disable Insider builds
+    1. Open **Settings**.
+    2. In the sidebar, select **About**.
+    3. Under **App**, disable **Receive insider builds**.
+2. Close Obsidian.
+3. Delete the file named `x.xx.xx.asar` (where the `x`s stand for numbers):
+   - Windows: `%APPDATA%\obsidian\obsidian-0.xx.xx.asar`
```

Comment:
```suggestion
   - Windows: `%APPDATA%\obsidian\obsidian-VERSION.asar`
```

Reply:
Another alternative is to only type the folder that contains the file, i.e. 

> Windows: `%APPDATA%\obsidian`.

Reply:
I've committed your first suggestion because it was easier to do, but both of your suggestions seem fine to me.

I think %APPDATA% with no explanation may confuse average users. I left it as-is because I don't have a good replacement, but hopefully it can be improved now or in the future.